apps/docs/src/development/packages/vitepress-theme.md

```diff
@@ -14,9 +14,14 @@ import packageJson from "../../../../../packages/vitepress-theme/package.json";
 
 </div>
 
-Custom [VitePress](https://vitepress.dev) theme that uses the onyx design system.
+Custom [VitePress](https://vitepress.dev) theme that uses the Schwarz Digits brand colors.
```

Comment:
```suggestion
Custom [VitePress](https://vitepress.dev) theme that uses the onyx brand colors.
```

Thats actually technical incorrect^^ We are not using the schwarz digits colors, we use the default onyx color palette.

I wouldn't describe it in that detail which colors we are using.